<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_node">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>节点2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>节点12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_hum">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>温度：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hum_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_realdata">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>实时监测数据：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_change">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>变化趋势</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_data">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>89</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>湿度</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>温度</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pm25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>光照度</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pm10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>大气压</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>风速</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>风向</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_ill">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>350</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>光照度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_atm">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>大气压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>风速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_direct">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>风向</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ill_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_atm_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>470</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_direct_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>510</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hum_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>时间：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pm25">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PM2.5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pm25_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ill_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>pm10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pm10_val">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="MyLineGraph" name="LineGraph">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_realdata_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>偏远地区生态环境监测系统</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineGraph</class>
   <extends>QGraphicsView</extends>
   <header>mylinegraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
